pay-api/src/pay_api/schemas/schemas/routing_slip_search_request.json

```diff
@@ -47,6 +47,16 @@
            "COMPLETED"
         ]
      },
+      "chequeReceiptNumber": {
+         "$id": "#/properties/receipts/items/properties/receiptNumber",
```

Comment:
properties/receipts/items/properties/receiptNumber to properties/receipts/items/properties/chequeReceiptNumber